ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -2262,23 +2262,23 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_round)
         onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/round.prototxt"));
     auto test_case = test::TestCase<TestEngine>(function);
 
-    test_case.add_input<float>({0.1f,
-                                0.5f,
-                                0.9f,
-                                1.2f,
-                                1.5f,
-                                1.8f,
-                                2.3f,
-                                2.5f,
-                                2.7f,
-                                -1.1f,
-                                -1.5f,
-                                -1.9f,
-                                -2.2f,
-                                -2.5f,
-                                -2.8f});
+    test_case.add_input<float>(
```

Comment:
Why did you change input and output values for this test?

Reply:
@ilyachur I split the test to two parts: `onnx_model_round` contains test data supported by limited version https://github.com/openvinotoolkit/openvino/pull/1605 (which we think is acceptable as temporary solution). Rest of test data was moved to `onnx_model_round_half_nearest_even` test which should PASS in this PR.

Reply:
Reported bug because of failing on IE: 35631